src/main/java/bio/terra/service/filedata/azure/tables/TableDependencyDao.java

```diff
@@ -90,23 +90,9 @@ public void deleteSnapshotFileDependencies(
       ListEntitiesOptions options =
           new ListEntitiesOptions().setFilter(String.format("snapshotId eq '%s'", snapshotId));
       PagedIterable<TableEntity> entities = tableClient.listEntities(options, null, null);
-      entities
-          .streamByPage()
-          .forEach(
-              entityChunk -> {
-                List<TableTransactionAction> batchEntities =
-                    entityChunk.getElements().stream()
-                        .map(
-                            entity ->
-                                new TableTransactionAction(
-                                    TableTransactionActionType.DELETE, entity))
-                        .collect(Collectors.toList());
-                logger.info(
-                    "Deleting snapshot {} file dependencies from {}",
-                    snapshotId,
-                    dependencyTableName);
-                tableClient.submitTransaction(batchEntities);
-              });
+      logger.info(
+          "Deleting snapshot {} file dependencies from {}", snapshotId, dependencyTableName);
+      entities.stream().forEach(tableClient::deleteEntity);
```

Comment:
Is there a deletion ordering we need to preserve?

Reply:
I don't think the order in which we delete these entries matters, since they're all individual rows in an Azure storage table